### ISSUE TITLE
Swift 3 and version 2.0.0

### ISSUE THREAD

**Issue by bartleby** [xxxAIRINxxx/ARNTransitionAnimator#6]
Hi, i do't know how to use version 2.0.0, can you help me?

**Comment by bartleby** [comment]
I used to use this code

``` Swift
import UIKit
import ARNTransitionAnimator

class MyAnimator {

    fileprivate var fromVC: UIViewController!
    fileprivate var gestureTargetView: UIView!
    fileprivate var toVC: UIViewController!
    fileprivate var animator: ARNTransitionAnimator!
}

extension MyAnimator {
    fileprivate func setupAnimator() {
        self.animator = ARNTransitionAnimator(operationType: .Present, fromVC: fromVC, toVC: toVC)
        self.animator.usingSpringWithDamping = 0.8
        self.animator.transitionDuration = 0.6
        self.animator.gestureTargetView = self.gestureTargetView
        self.animator.interactiveType = .Present

        // Present

        self.animator.presentationBeforeHandler = { [unowned self] containerView, transitionContext in

            self.animator.presentationCancelAnimationHandler = { [unowned self] containerView in }

            self.animator.presentationAnimationHandler = { [unowned self] containerView, percentComplete in }

            self.animator.presentationCompletionHandler = { [unowned self] containerView, completeTransition in }
        }

        // Dismiss

        self.animator.dismissalBeforeHandler = { [unowned self] containerView, transitionContext in

            self.animator.dismissalCancelAnimationHandler = { [unowned self] containerView in }

            self.animator.dismissalAnimationHandler = { [unowned self] containerView, percentComplete in }

            self.animator.dismissalCompletionHandler = { [unowned self] containerView, completeTransition in }
        }

        self.toVC.transitioningDelegate = self.animator
    }
}
```

how to use now?

**Comment by xxxAIRINxxx** [comment]
Hi.
This is implemented in ARNTransitionAnimator 2.0.0 (Swift 3)
Please try to reference.

https://github.com/xxxAIRINxxx/SoundCloudTransition
